app/gui/view/debug_scene/interface/src/lib.rs

```diff
@@ -338,6 +338,7 @@ pub fn expression_mock() -> Expression {
     Expression { pattern, code, whole_expression_id, input_span_tree, output_span_tree }
 }
 
+// TODO[ao] This expression mocks results in panic. If you want to use it, please fix it first.
```

Comment:
I don't quite understand this sentence, and also I'm not sure if we should leave in code TODOs not attached to a created issue in pivotal.